4.DZ/date.hpp

```diff
@@ -0,0 +1,22 @@
+#include <iostream>
+
+namespace DZ4 {
+
+struct Date {
+	int day;
+	int month;
+	int year;
+};
+
+void run() {
+	Date date;
+	char slash = '/';
+	std::cout << "Enter the date in the format day/month/year: " << std::endl;
+	std::cin >> date.day >> slash >> date.month >> slash >> date.year;
+	if (date.day < 31 && date.month < 13) {
```

Comment:
if I set `-1000` to `date.day`, any handler for this case?
if I set `-1000` to `date.month`, any handler for this case?